Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionProcessId.cs

```diff
@@ -21,6 +21,16 @@ public static IdentityDeletionProcessId Generate()
         return new IdentityDeletionProcessId(PREFIX + randomPart);
     }
 
+    public static bool IsValid(string stringValue)
+    {
+        return UTILS.IsValid(stringValue);
+    }
```

Comment:
Is it possible to write an ArchUnit test that asserts that a type inheriting from StronglyTypedId has a static method called `IsValid` with a single parameter of type `string`?

Reply:
What about this?